specification/servicenetworking/ServiceNetworking.Management/client.tsp

```diff
@@ -46,3 +46,11 @@ using Microsoft.ServiceNetworking;
   "TrafficControllerProvisioningState",
   "csharp"
 );
+@@clientName(Versions.v2023_11_01,
+  "$DO_NOT_NORMALIZE$V2023_11_01",
+  "javascript"
+);
+@@clientName(Versions.v2025_05_01,
+  "$DO_NOT_NORMALIZE$V2025_05_01",
+  "javascript"
```

Comment:
@kazrael2119 could you confirm if two version impact?

Reply:
yes
![image](https://github.com/user-attachments/assets/f4d8dd9f-7514-42ff-a7a0-6c20c439202f)